packages/components/components/elvis-accordion/src/react/elvia-accordion.tsx

```diff
@@ -56,20 +56,6 @@ export const Accordion: FC<AccordionProps> = ({
     isOpen ? 'unset' : type === 'overflow' ? 'unset' : 'hidden',
   );
 
-  const accordionRef = useRef<HTMLDivElement>(null);
-
-  /** Start outline listener */
-  useEffect(() => {
-    if (accordionRef?.current) {
-      outlineListener(accordionRef.current);
-    }
-    return () => {
-      if (accordionRef?.current) {
-        outlineListener(accordionRef.current, true);
-      }
-    };
-  }, []);
```

Comment:
This added the class `ewc-no-outline`, which doesn't actually do _anything_ anymore. I think it used to do something, once upon a time